Sources/Base/OAuth2KeychainAccount.swift

```diff
@@ -59,6 +59,20 @@ struct OAuth2KeychainAccount: KeychainGenericPasswordType {
 		accessGroup = oauth2.keychainAccessGroup
 		data = inData
 	}
+	
+	/**
+	Designated initializer.
+	- parameter _serviceName:  The OAuth2 instance Keychain Entry Name
+	*/
+	
+	init(oauth2: OAuth2Securable, account: String, data inData: [String: Any] = [:],_serviceName:String) {
```

Comment:
What we will be passing in _serviceName params?

Reply:
`/**
	Designated initializer.
	- parameter _serviceName:  The OAuth2 instance Keychain Entry Name
	*/`